sdk/easm/defender-easm-rest/review/defender-easm.api.md

```diff
@@ -318,7 +331,7 @@ export interface CookieOutput {
 }
 
 // @public
-function createClient(endpoint: string, subscriptionId: string, resourceGroupName: string, workspaceName: string, credentials: TokenCredential, options?: ClientOptions): EasmClient;
+function createClient(endpointParam: string, credentials: TokenCredential, { apiVersion, ...options }?: EasmClientOptions): EasmClient;
```

Comment:
this is a known service typespec breaking, where we can't using versioning to express in typespec, see https://github.com/microsoft/typespec/issues/4317